heron/packing/src/java/com/twitter/heron/packing/PackingUtils.java

```diff
@@ -160,4 +167,12 @@ public static boolean isValidInstance(Resource instanceResources,
     return currentContainers;
   }
 
+  public static long increaseBy(long value, int paddingPercentage) {
+    return value + (paddingPercentage * value) / 100;
+  }
+
+  public static double increaseBy(double value, int paddingPercentage) {
```

Comment:
@billonahill I would prefer it to keep it outside in the algorithm logic and not in the utils. Other algos could implement ceiling or sth else for cpu.